tools/jit/templates/aten_dispatch.cpp

```diff
@@ -92,12 +92,14 @@ ConstructorsMap constructors = {
 
 std::string getDescriptor(jit::Node* n) {
   std::stringstream s;
-  s << n->kind().toString();
+  // TODO: Only ATen operators get JIT dispatchers.  Test if this is the case
+  // and complain if it's not.
```

Comment:
This should be simple to do now, right?

Reply:
The more important question is if it's true :) Test added.